### DISCUSSION TITLE
Audiogridder vst plugin Unusually high cpu usage

### DISCUSSION THREAD

**Discussion by FrancescoAree** [apohl79/audiogridder#1109] [Help]
Hi, guys thank's for having me , hope somebody can help me,  i've experience some issue,  i'm using the audiogridder server on a old  windows , and the plug-in in a macbook pro retina 13inch with bigsur, updated just some days ago, when i was with catalina 15.7 i didn't experience any issues work very well with ableton with the beta vrs 15, now when i put the plugin  "audiogridder vst" inside a channel the cpu of ableton  goes around 200, i will add a video for show you the problem, it's also difficult to put me in contact with the company the plug-in is great, would love if someone can help me to resolve this problem. 
both same wifi and and also connected with a LANcable. 

you can also add me in IG: Francesco_Aree  i really appreciate your help

**Comment by baaccs** [comment]
It is probably a interface or audio driver issue.  I am trying it for the first time instead of Sound grid because I love the idea of it working with most plugins.   I was having an issue of sorts and for some reason it doesn't like something with the Behringer U-Phoria 1820.  The interface works flawlessly without Audio Gridder but after about 3 to 5 plugins it would start overloading the DAW computer but the server was fine.  I tried with just the internal windows audio output, and it works great.  WEIRD!!!  Just using a a 2gig 2 Core i3 that's an older computer and it took a project I was at 90% DSP to 18% DSP WOW, loving it, hopefully can find out what is causing the issue when using the interface.

**Comment by apohl79** [comment]
> both same wifi and and also connected with a LANcable

you should make sure you are not using the wifi connection for AG as this would have a big impact on latency and might explain your high meter in ableton.

**Comment by PierreMConsulting** [comment]
Hi There! I've been running AG Beta 9 on Catalina for a while and did not had that kind of issue. 

I've recently upgraded my MacOS to Monterey and my Windows server to windows 11 and I've started to have the CPU spike issue with Beta 9. 

I upgraded to beta 15 and it did not help. I'm using an ethernet Gigabit network with a pro central switch between the Mac and the PC, using high quality cable.

I can work for hours without any glitch but when it starts to glitch I usually have to reboot the two computers. Sometimes one reboot is enough sometimes it's not. The AG client monitor is acting like it's loosing the connection and reconnecting.

A few things I've tested if it can help:

- Restarting the server does not help
- Closing the DAW session and opening it again does not help
- Shutting down my audio interface chain does not help
- Restarting the Mac helps. Once I did that the server side was frozen when I tried to load the session so I had to kill the process and restart the server

Update: it happened a second time and restarting the Mac allowed to get back to a working session after reboot even without restarting the AG Server.

Happy to help.

Best,

Max

**Comment by PierreMConsulting** [comment]
@apohl79 Feel free to reach out if you need any trace or debug. In the meantime I've resetted my Mac network settings (deleted the configuration files and restarted the OS) in case it could be related to the OS migration process and will update the discussion here as soon as the issue reproduces.

Best,

Max

**Comment by PierreMConsulting** [comment]
Hi @apohl79,

After having done a reset of my mac network settings configuration files the issue still persist.

It has definitely been introduced with the OS migration as I never had it it on Catalina.

Best,

Max

**Comment by PierreMConsulting** [comment]
Hi @apohl79,

The issue just reproduced and rebooting the Mac client fixed it. 

So far this is the only workaround I've found.

It's working fine for hours then suddenly starts to reconnect/disconnect endlessly which generates CPU spikes in the DAW which makes the session impossible to play.

Please find the client logs following this link: [here](https://www.dropbox.com/s/xlqtngom4uu3vyh/FX.zip?dl=0)
